From e67ec9050d552ff0c66cc8bc9571ae50e391f33d Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Wed, 24 Oct 2012 17:02:41 +0800
Subject: [PATCH 084/602] ENGR00230969 SABRESD_6DQ: OTA: change server address
 for jellybean.

Change jellybean OTA server address since daily build is on another server.

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 imx6/etc/ota.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/etc/ota.conf b/imx6/etc/ota.conf
index a44a432..f8af05c 100644
--- a/imx6/etc/ota.conf
+++ b/imx6/etc/ota.conf
@@ -1,2 +1,2 @@
-server=10.192.224.88
+server=10.192.224.178
 port=10888
-- 
1.8.0

